eng/Versions.props

```diff
@@ -193,17 +193,17 @@
     <MicrosoftCodeAnalysisPublicApiAnalyzersVersion>3.3.0</MicrosoftCodeAnalysisPublicApiAnalyzersVersion>
     <MicrosoftCssParserVersion>1.0.0-20200708.1</MicrosoftCssParserVersion>
     <MicrosoftIdentityModelClientsActiveDirectoryVersion>3.19.8</MicrosoftIdentityModelClientsActiveDirectoryVersion>
```

Comment:
>3.19.8< [](start = 57, length = 7)

Where is this used? Any reason this is not using M.I.Client instead?

Reply:
It doesn't seem to be used anymore. https://github.com/dotnet/aspnetcore/pull/31484 removes it.